promtail/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ -z "$CHARON_LOKI_ADDRESSES" ]
+then
+  echo "\$CHARON_LOKI_ADDRESSES variable is empty"
```

Comment:
Would we consider exiting with a non zero, so promtail container doesn't start? or will /usr/bin/promtail handle that for us i wonder? 

(example of setting the script to fail if a command fails for convenience: https://github.com/pcaversaccio/white-hat-frontrunning/blob/main/go.sh#L15)